README.md

```diff
@@ -826,6 +848,12 @@ __Arguments__
 - __ignore_blank_headers (bool, optional)__:
         When True, ignores all columns
         that have blank headers. Defaults to False.
+- __ignore_listed_headers (List[str], optional)__:
+        When passed, ignores all columns
+        that listed in the given list
+- __ignore_not_listed_headers (List[str], optional)__:
+        When passed, ignores all columns
+        that NOT listed in the given list
```

Comment:
Perhaps "When passed, ignores all columns that are NOT included in the given list" is more clear